docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+---
+version: "2"
+services:
+  homer:
+    image: b4bz/homer
+    container_name: homer
+    volumes:
+      - /your/local/assets/:/www/assets
+    ports:
+      - 8080:8080
+    #user: "1000:1000" # UID:GID
```

Comment:
Does not work as intended. It still tries to set the GID to "911", but the files belong to 1000:1000 already.
Results into the following output:

> homer    | chown: /www/assets/config.yml: Operation not permitted
homer    | chown: /www/assets/config.yml.dist: Operation not permitted
homer    | chown: /www/assets/config.yml.dist.sample-sui: Operation not permitted
homer    | chown: /www/assets/icons/apple-touch-icon.png: Operation not permitted
homer    | chown: /www/assets/icons/favicon-16x16.png: Operation not permitted
homer    | chown: /www/assets/icons/android-chrome-maskable-512x512.png: Operation not permitted
homer    | chown: /www/assets/icons/favicon-32x32.png: Operation not permitted
homer    | chown: /www/assets/icons/apple-touch-icon-76x76.png: Operation not permitted
homer    | chown: /www/assets/icons/apple-touch-icon-180x180.png: Operation not permitted
homer    | chown: /www/assets/icons/android-chrome-512x512.png: Operation not permitted
homer    | chown: /www/assets/icons/android-chrome-maskable-192x192.png: Operation not permitted
homer    | chown: /www/assets/icons/safari-pinned-tab.svg: Operation not permitted
homer    | chown: /www/assets/icons/apple-touch-icon-60x60.png: Operation not permitted
homer    | chown: /www/assets/icons/msapplication-icon-144x144.png: Operation not permitted
homer    | chown: /www/assets/icons/apple-touch-icon-152x152.png: Operation not permitted
homer    | chown: /www/assets/icons/android-chrome-192x192.png: Operation not permitted
homer    | chown: /www/assets/icons/mstile-150x150.png: Operation not permitted
homer    | chown: /www/assets/icons/apple-touch-icon-120x120.png: Operation not permitted
homer    | chown: /www/assets/icons: Operation not permitted
homer    | chown: /www/assets/icons: Operation not permitted
homer    | chown: /www/assets/manifest.json: Operation not permitted
homer    | chown: /www/assets/tools/sample2.png: Operation not permitted
homer    | chown: /www/assets/tools/sample.png: Operation not permitted
homer    | chown: /www/assets/tools: Operation not permitted
homer    | chown: /www/assets/tools: Operation not permitted
homer    | su-exec: setgroups(911): Operation not permitted

Reply:
Does work with the following lines in the docker-compose.yml:
>    environment:
>      - UID=1000
>      - GID=1000